com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs

```diff
@@ -1436,6 +1436,8 @@ void RegisterRenderingDebug()
                 widgetList.Add(new DebugUI.BoolField { displayName = "XR single-pass test mode", getter = () => data.xrSinglePassTestMode, setter = value => data.xrSinglePassTestMode = value });
             }
 
+            //widgetList.Add(new DebugUI.BoolField { displayName = "Enable Render Graph", getter = () => HDRenderPipeline.currentPipeline.IsRenderGraphEnabled(), setter = value => HDRenderPipeline.currentPipeline.EnableRenderGraph(value) });
```

Comment:
Do we want to keep this commented for the time being or is it a leftover?

Reply:
Yeah, kind of dirty but this way Render Graph is not available in the debug menu and I don't really want to have to rewrite this everytime I (or someone else) need to work on it.